### PR TITLE
#12: create and style sec4-sec5 card caseStudy and input

### PR DESCRIPTION
**Pull request by Lanng21**
image
image

### CODE REVIEW

File: src/stylesheets/components/_common.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @use "../abstracts/" as *;
 @forward "slider";
-
+@forward "proment-sec";
```
Comment (baonguyen26) [inline src/stylesheets/components/_common.scss]: Add break line here

File: src/stylesheets/pages/home/_send-mail-sec.scss [hunk 2]
```diff
@@ -0,0 +1,160 @@
+@use "../../abstracts/" as *;
+#send-email-sec {
```
Comment (baonguyen26) [inline src/stylesheets/pages/home/_send-mail-sec.scss:2]: Break line, pls

File: src/stylesheets/pages/home/_send-mail-sec.scss [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    justify-content: center;
+    position: relative;
+    padding: 70px 0;
+
+    .cards {
+      display: flex;
+      margin-top: 100px;
+
+      .card {
+        width: 360px;
```
Comment (baonguyen26) [inline src/stylesheets/pages/home/_send-mail-sec.scss]: Fixed width?

File: src/stylesheets/pages/home/_send-mail-sec.scss [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+        margin-bottom: auto;
+      }
+
+      .card2 {
+        margin-top: auto;
+      }
+    }
+
+    .heading {
+      margin: 25px 150px;
```
Comment (baonguyen26) [inline src/stylesheets/pages/home/_send-mail-sec.scss]: Margin value too large

File: src/stylesheets/pages/home/_send-mail-sec.scss [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+        @include tablet {
+          font-size: 28px;
+          line-height: 36px;
+          font-weight: 900;
+        }
+      }
+
+      .input-field {
+        input {
+          width: 410px;
```
Comment (baonguyen26) [inline src/stylesheets/pages/home/_send-mail-sec.scss]: same

**Review by haunguyen15-tt — APPROVED** [review]

**Pull request merged**